scripts/validate_docstrings.py

```diff
@@ -527,8 +527,35 @@ def validate_one(func_name):
                                             err.message))
 
     if doc.is_function_or_method:
-        if not doc.returns and "return" in doc.method_source:
-            errs.append('No Returns section found')
+        if not doc.returns:
+            if "return" in doc.method_source:
+                errs.append('No Returns section found')
+        else:
+            returns_errs = []
+            if len(doc.returns) == 1 and doc.returns[0][1]:
+                returns_errs.append('The first line of the Returns section '
+                                    'should contain only the type, unless '
+                                    'multiple values are being returned.')
+            missing_desc, missing_cap, missing_period = False, False, False
```

Comment:
`missing_desc = missing_cap = missing_period = False`

Reply:
With that said is this even required? It seems like you could simplify things without the `or` statements below

Reply:
the `or` statement are needed to print at most one message of each kind.
as the error messages do not specify the variable names, I assumed it would be clearer to avoid printing the same message twice (or even more).